### PR TITLE
Update User guide

### PR DESCRIPTION
**Pull request by beesaycheese**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -151,9 +152,10 @@ Some of the parameters have restrictions and they are stated below.
 * `QUANTITY` has to be a whole number between 0 and 2,000,000 inclusive.
 * `ITEM_NAME` has to be
 * `ITEM_PRICE` has to be
-* `DAY` has to be
-* `MONTH` has to be
-* `YEAR` has to be
+* `YEAR` has to be be within 2000 and the current year.
```
Comment (whyjayxp) [inline docs/UserGuide.adoc]: Can be more specific: eg. whole number, 4 digit number etc

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -151,9 +152,10 @@ Some of the parameters have restrictions and they are stated below.
...
+* `MONTH` has to be within 1 to 12.
+* `DAY` has to be within 1 to 31 (Special case: 29, 30 and 31 are only vaild if it exists in the month of a specified
```
Comment (whyjayxp) [inline docs/UserGuide.adoc]: Typo: "valid"; also, phrasing seems a little awkward, consider: (provided it exists in the desired month and year).

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -610,16 +612,54 @@ The Order Panel will also be updated and you can see from the image below that t
...
 
-Examples:
+****
+*Format*: `bill` +
+*Shortcut*: `b`
+****
+
+[IMPORTANT]
+This command is irreversible. Once the command is executed, all orders and the table occupancy will be cleared to allow
+new customers to order at the table. The money earned from the bill will automatically be added in the
```
Comment (whyjayxp) [inline docs/UserGuide.adoc]: new customers to *use* the table. The money earned from the bill will automatically be added *to*

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -610,16 +612,54 @@ The Order Panel will also be updated and you can see from the image below that t
...
+RestOrRant's daily revenue statistics.
```
Comment (whyjayxp) [inline docs/UserGuide.adoc]: don't need to mention "daily"

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -679,54 +719,142 @@ Examples:
...
-[[Revenue]]
-==== Getting the total revenue : `revenue`
+The function of the side and main panels are as follows:
+
+* *Item Panel*: This panel shows you the list of menu items sorted by its popularity. This is meant to help you decide
+what ingredients to order to cater to the demands of the popular dishes.
+* *Revenue Panel*: This panel displays the revenue from the past 30 years, months and days for you to study the revenue trends.
+
+[NOTE]
+By default the Statistics Mode displays revenue from the past 30 days in the Revenue pPnel unless you execute
```
Comment (whyjayxp) [inline docs/UserGuide.adoc]: "up to"; "Panel"

File: docs/UserGuide.adoc [hunk 6]
```diff
@@ -679,54 +719,142 @@ Examples:
...
+<<Yearly, `yearly`>>, <<Monthly, `monthly`>> or <<Daily, `daily`>> commands to change its display respectively.
 
-Returns the total revenue earned by the restaurant by the stated year, month or day. +
-Format: `revenue [d/DAY] [m/MONTH] [y/YEAR]`
+Here are the commands that you can use in this mode to efficiently study the statistical data of your restaurant.
```
Comment (whyjayxp) [inline docs/UserGuide.adoc]: the word efficiently here feels weird

File: docs/UserGuide.adoc [hunk 7]
```diff
@@ -679,54 +719,142 @@ Examples:
...
 
 image::SMUi.png[width="790"]
 
-[[Revenue]]
-==== Getting the total revenue : `revenue`
+The function of the side and main panels are as follows:
+
+* *Item Panel*: This panel shows you the list of menu items sorted by its popularity. This is meant to help you decide
+what ingredients to order to cater to the demands of the popular dishes.
+* *Revenue Panel*: This panel displays the revenue from the past 30 years, months and days for you to study the revenue trends.
```
Comment (whyjayxp) [inline docs/UserGuide.adoc]: years, months or days, depending on the view that you have selected. This allows you to study trends in your restaurant's revenue.

File: docs/UserGuide.adoc [hunk 8]
```diff
@@ -679,54 +719,142 @@ Examples:
...
+
+[width="100%",cols="5%,95%", grid=none]
+|===
+ifdef::env-github[| :white_check_mark: a|]
+ifndef::env-github[a| icon:check[role="green", size="2x"] a|]
+You will see a success message in the Results Display and the Revenue Panel changes to display up to 30 monthly
+revenue starting from the most recent record.
+
+
+a| a| Notice that the first monthly revenue is the most recent month recorded. This saves you from the trouble of
```
Comment (whyjayxp) [inline docs/UserGuide.adoc]: saves you the trouble of

File: docs/UserGuide.adoc [hunk 9]
```diff
@@ -679,54 +719,142 @@ Examples:
...
+
+|===
+
+[[Revenue]]
+==== Getting the total revenue : `revenue`
+
+If you want to obtain the revenue earned in a particular year, month or day, you can obtain it with this command.
+
+****
+*Format*: `revenue [ y/YEAR [m/MONTH [d/DAY] ] ]` +
```
Comment (whyjayxp) [inline docs/UserGuide.adoc]: haha this is confusing, consider breaking into 4 examples

File: docs/UserGuide.adoc [hunk 10]
```diff
@@ -746,7 +874,7 @@ The list of items ordered by dining customers at a table
 The list of items available to be ordered by dining customers at the restaurant
 
 [[bill]] Bill::
-The amount of money that dining customers have to pay based on their order
+The amount of money that dining customers have to pay based on their orders
```
Comment (whyjayxp) [inline docs/UserGuide.adoc]: Nope, "order" is correct. Refer to the definition of order above this

File: docs/UserGuide.adoc [hunk 11]
```diff
@@ -840,6 +968,12 @@ These commands can only be used in Statistics Mode.
 [width="90%",cols="^25%,^15%,30%,30%",options="header",]
 |=======================================================================
 ^|Command ^|Shortcut ^|Usage ^|Example
-a|<<Revenue, `revenue`>> a|`*r*` a|`revenue [d/DAY] [m/MONTH] [y/YEAR]` a|`revenue d/1 m/1 y/2019` +
+a|<<Yearly, `yearly`>> a|`*y*` a|`yearly` a|
+Displays up to 30 revenue from recent years in the Revenue Panel
```
Comment (whyjayxp) [inline docs/UserGuide.adoc]: awkward phrasing, consider: "Displays the revenue from up to the past 30 years in the Revenue Panel"

File: docs/UserGuide.adoc (hunk 10 above)
Comment (beesaycheese) [reply docs/UserGuide.adoc]: okk

**Pull request merged**